extensions/everything-search/package.json

```diff
@@ -2,58 +2,221 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "everything-search",
   "title": "Everything",
-  "description": "Search multiple Everything ETP/FTP servers from Raycast",
+  "description": "Search files using Everything on Windows.",
   "icon": "extension-icon.png",
-  "platforms": ["Windows"],
   "author": "anastasiy_safari",
+  "contributors": [
+    "laurynas_gailius"
+  ],
+  "pastContributors": [
+    "doug_fernando"
+  ],
   "license": "MIT",
   "commands": [
+    {
+      "name": "search-everything",
+      "title": "Search Files",
+      "description": "Search files using Everything",
+      "mode": "view"
+    },
     {
       "name": "evy",
-      "title": "Search Files by Mask",
+      "title": "Search Files on Servers",
       "subtitle": "Everything",
-      "description": "Find Files from Everything by Mask",
+      "description": "Find Files from Everything ETP/FTP servers by Mask",
       "mode": "view",
+      "disabledByDefault": true,
```

Comment:
Is there any point of disabling it by default? We tend to do that when there are a lot of commands but here there are only 2 so it's not hard for users to disable it themselves if they want to